docs/docs/clients/server-side.md

```diff
@@ -1824,7 +1824,7 @@ router.get('/', function (req, res, next) {
 <TabItem value="php" label="PHP">
 
 ```php
-$flagsmith = (new Flagsmith(TOKEN));
+$flagsmith = (new Flagsmith(FLAGSMITH_SERVER_SIDE_ENVIRONMENT_KEY));
```

Comment:
Shouldn't this be in quotes like the rest of the listings of the keys?

Reply:
Ooh well spotted